src/librustdoc/clean/blanket_impl.rs

```diff
@@ -17,7 +17,7 @@ impl<'a, 'tcx> BlanketImplFinder<'a, 'tcx> {
         let param_env = self.cx.tcx.param_env(item_def_id);
         let ty = self.cx.tcx.type_of(item_def_id);
 
-        debug!("get_blanket_impls({:?})", ty);
+        trace!("get_blanket_impls({:?})", ty);
```

Comment:
I can drop this whole commit if you like, it's not necessary, just made debugging easier.

Reply:
Debugging is always a good thing.